core/schema_utils_test.py

```diff
@@ -890,6 +890,18 @@ def test_has_length(self) -> None:
         self.assertFalse(has_length(['efg'], 2))
         self.assertFalse(has_length(['efg', 'ghj', 'huij'], 2))
 
+    def test_should_end_with_validator(self) -> None:
+        """Tests if a given filename has an allowed audio
+        extension.
+        """
+        should_end_with = schema_utils.get_validator(
+            'should_end_with')
+        self.assertTrue(should_end_with('abcxyz', ['XYZ', 'efg']))
+        self.assertTrue(should_end_with('abcXyZ', ['efg', 'XyZ'], True))
+
+        self.assertFalse(should_end_with('abcdef', ['xyz', 'efg']))
+        self.assertFalse(should_end_with('abcXYZ', ['xyz', 'efg'], True))
```

Comment:
Why is this assertFalse?

Reply:
For the first assertFalse (line 902), since the _abcdef_  doesn't end with _xyz_ or _efg_ the return value should be false. For the second assert, since match_case is True, _XYZ_ will not match with _xyz_ and the return value would be false.

Reply:
I see thanks for the explanation, make sure to add another check similar to 2nd assert with match_case=False

Reply:
Done.

Reply:
Was this fixed?